charts/devtron/templates/workflow.yaml

```diff
@@ -1201,9 +1201,9 @@ spec:
           value: v1
         {{- end }}
         {{- if $.Values.workflowController.IMDSv2Enforced }}
-        image: {{ $.Values.workflowController.image }}
+        image: {{ include "common.image" (dict "component" $.Values.workflowController "global" $.Values.global  ) }}
         {{- else }}
-        image: quay.io/argoproj/workflow-controller:v3.0.7
+        image: {{ include "common.image" (dict "component" $.Values.workflowController.IMDSv1 "global" $.Values.global ) }}
```

Comment:
component is set till IMDSv1 and under IMDSv1 there is no image key, IMDSv1 itself is key, is this working in templating?

Reply:
As the default configuration for IMDSv2 is set to true, so that the false case has been overlooked during the template